README.md

```diff
@@ -226,11 +225,11 @@ Alternatively, you can block until the rate limit is reset by using the `context
 repos, _, err := client.Repositories.List(context.WithValue(ctx, github.SleepUntilPrimaryRateLimitResetWhenRateLimited, true), "", nil)
 ```
 
-You can use [go-github-ratelimit](https://github.com/gofri/go-github-ratelimit) to handle
+You can use [gofri/go-github-ratelimit](https://github.com/gofri/go-github-ratelimit) to handle
 secondary rate limit sleep-and-retry for you.
 
-Learn more about GitHub secondary rate limiting at
-https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28#about-secondary-rate-limits .
+Learn more about GitHub secondary rate limiting in the
```

Comment:
```suggestion
Learn more about GitHub secondary rate limiting in
```

Reply:
WHUPS!!!  It should be Go 1.22 and Go 1.23, so:

Maybe "go-github tests against Go version 1.22 and greater" here?